cycode/cli/files_collector/sca/sbt/restore_sbt_dependencies.py

```diff
@@ -0,0 +1,30 @@
+import os
+from typing import List, Optional
+
+import click
+
+from cycode.cli.files_collector.sca.base_restore_dependencies import BaseRestoreDependencies
+from cycode.cli.models import Document
+
+SBT_PROJECT_FILE_EXTENSIONS = ['sbt']
+SBT_LOCK_FILE_NAME = 'build.sbt.lock'
+
+
+class RestoreSbtDependencies(BaseRestoreDependencies):
+    def __init__(self, context: click.Context, is_git_diff: bool, command_timeout: int) -> None:
+        super().__init__(context, is_git_diff, command_timeout)
```

Comment:
looks like it is blank constructor which calls constructor from the base. no need to override it here

Reply:
fixed